src/assets/CollectibleDetectionController.ts

```diff
@@ -249,7 +251,10 @@ export class CollectibleDetectionController extends BaseController<
     });
 
     onNetworkStateChange(({ provider }) => {
-      this.configure({ networkType: provider.type });
+      this.configure({
+        networkType: provider.type,
+        chainId: provider.chainId as CollectibleDetectionConfig['chainId'],
```

Comment:
Nit: Ideally we'd parse this here to guarantee that the chain ID is the expected type, rather than cast. Or propagate this new chain ID type into the network controller as well. I guess we can do that later.